gkShell/config/konfig-local.php

```diff
@@ -0,0 +1,52 @@
+<?php
```

Comment:
So bad :disappointed: , this is a copy of file `configs/konfig-local.tmpl.php`

And they already started to differ! (`unset($GOOGLE_RECAPTCHA_PUBLIC_KEY);` part bellow is not there :/

And as this file is committed, it will open risks to have commits including this file too by inadvertence. Isn't it possible to have `gkShell` INSTALL instruction asking user to copy template from `configs/konfig-local.tmpl.php`?

Reply:
will make "live copy + gitignore" as for apache/mariadb to avoid duplicate

Reply:
done

Reply:
need to add section in readme